database/migrations/2022_02_22_181909_create_meetups_table.php

```diff
@@ -13,7 +13,6 @@ public function up(): void
             $table->id();
             $table->foreignId("user_id")->constrained()->onDelete("cascade");
             $table->foreignId("organization_id")->nullable()->constrained()->onDelete("set null");
-            $table->foreignId("speaker_id")->nullable()->constrained()->onDelete("set null");
```

Comment:
Please do not modify existing migrations. You can apply this change in `2022_04_22_103521_create_meetup_speaker_table.php`

Reply:
SQLite doesn't support adding/dropping foreign keys to/from existing tables.
https://laravel.com/docs/9.x/migrations#toggling-foreign-key-constraints

Reply:
Oh no.